tools/lsp/ui/components/widgets/color-widget.slint

```diff
@@ -50,58 +51,33 @@ export component ColorWidget inherits GridLayout {
         if !self.has-focus {
             apply-value();
         }
+
+        // Floating editors need to know the current property details. Call the update if something changes.
+        if WindowManager.showing-color-picker && WindowManager.current-property-information.name == root.property-name {
```

Comment:
Could you make the condition a bit stronger?

E.g. a `property-name` of `x` is widely used by many elements, but you want to only apply changes to the `x` property of one element.